src/libs/fsdrv/lv_fs_memfs.c

```diff
@@ -97,23 +90,23 @@ void lv_fs_memfs_init(void)
 
     /*Set up fields...*/
     fs_drv.letter = LV_FS_MEMFS_LETTER;
-    fs_drv.cache_size = UINT32_MAX; //LV_FS_MEMFS_CACHE_SIZE;
+    fs_drv.cache_size = LV_FS_CACHE_FROM_BUFFER;
 
     fs_drv.open_cb = fs_open;
     fs_drv.close_cb = fs_close;
     fs_drv.read_cb = fs_read;
-    fs_drv.write_cb = NULL; // fs_write;
+    fs_drv.write_cb = NULL;
     fs_drv.seek_cb = fs_seek;
     fs_drv.tell_cb = fs_tell;
 
-    fs_drv.dir_close_cb = NULL; // fs_dir_close;
-    fs_drv.dir_open_cb = NULL; // fs_dir_open;
-    fs_drv.dir_read_cb = NULL; // fs_dir_read;
+    fs_drv.dir_close_cb = NULL;
+    fs_drv.dir_open_cb = NULL;
+    fs_drv.dir_read_cb = NULL;
 
     lv_fs_drv_register(&fs_drv);
 }
 
-void lv_fs_make_path_ex(lv_fs_path_ex_t* path, const uint8_t* buf, uint32_t size)
+void lv_fs_make_path_ex(lv_fs_path_ex_t* path, void* buf, uint32_t size)
```

Comment:
I think it makes sens to use `const void *` here, as we don't write anything into the buffer.

Reply:
```suggestion
void lv_fs_make_path_ex(lv_fs_path_ex_t* path, const void* buf, uint32_t size)
```

Compilation is failing here with this message:

```
lvgl/src/libs/fsdrv/lv_fs_memfs.c:109:6: error: conflicting types for 'lv_fs_make_path_ex'
void lv_fs_make_path_ex(lv_fs_path_ex_t* path, void* buf, uint32_t size)
     ^
lvgl/src/core/../draw/../misc/lv_fs.h:182:6: note: previous declaration is here
void lv_fs_make_path_ex(lv_fs_path_ex_t* path, const void* buf, uint32_t size);
```